api/server.go

```diff
@@ -628,6 +629,31 @@ func (s *server) pubkeyHostHandler(jc jape.Context) {
 	jc.Encode(hosts[0])
 }
 
+func (s *server) hostsListHandler(jc jape.Context) {
+	var params explorer.HostQuery
+	if jc.Decode(&params) != nil {
+		return
+	}
+
+	limit := uint64(100)
+	offset := uint64(0)
+	if jc.DecodeForm("limit", &limit) != nil || jc.DecodeForm("offset", &offset) != nil {
+		return
+	}
+
+	dir := explorer.HostSortAsc
```

Comment:
`nit` would reverse the order and return the newer hosts first?

Reply:
I think this is fine. Imo ascending is the more natural default for most fields we could sort by here. Also means the results for lower offsets on the default query don't change as much since newer hosts come last.